Linux/sgx/test_app/app/TestApp.cpp

```diff
@@ -153,11 +156,21 @@ void print_error_message(sgx_status_t ret)
             break;
         }
     }
-    
+
```

Comment:
If we submit this upstream, they may not want the unrelated whitespace changes. Although if we submit it upstream, this should probably be split to a separate sample app.

Reply:
I would create a separate application in any case so we it is easier to integrate.